src/main/java/com/vinyl/service/Validation.java

```diff
@@ -0,0 +1,23 @@
+package com.vinyl.service;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import com.vinyl.model.User;
+import com.vinyl.service.exception.BadRequestException;
+
+public class Validation {
+
+	public static final String nameRegex = "^[A-Z][a-zA-Z]+$";
+	public static final Pattern VALID_NAME_REGEX = Pattern.compile(nameRegex);
```

Comment:
it is a Pattern, not a regex VALID_NAME_PATTERN

Reply:
fixed